example-config.yaml

```diff
@@ -1,28 +1,97 @@
+#Sample Test Case for Ansible Tool Functionality
+
 ---
+organization_name: "Ansible Networking"       
+
+workflow_name: "tests.yml"                    
+
 orgs:
   - name: ansible-collections
-    repos:
+    ci_and_bug_repos:                         
       - name: cisco.nxos
       - name: cisco.ios
       - name: cisco.iosxr
       - name: arista.eos
-      - name: vyos.vyos
       - name: junipernetworks.junos
       - name: cisco.asa
       - name: ansible.netcommon
+      - name: ansible.snmp
+      - name: ansible.security
+      - name: ansible.network
+      - name: ansible.utils
+      - name: ibm.qradar
+      - name: trendmicro.deepsec
+      - name: splunk.es
+    bug_specific_repos:                     
+      - name: ansible.scm
+      - name: vyos.vyos
+      - name: frr.frr
+      - name: openvswitch.openvswitch
+      - name: community.yang
   - name: ansible
-    repos:
-      - name: ansible
-        labels:
-          - networking
-          - needs_triage
+    bug_specific_repos:
       - name: pylibssh
-maintainers:
-  - name: John Doe
-    email: john@doe.localhost
-  - name: Jane Doe
-    email: jane@doe.localhost
+  - name: redhat-cop
+    ci_and_bug_repos:
+      - name: network.interfaces
+      - name: network.bgp
+      - name: network.base
+      - name: network.telemetry
+      - name: network.vpn
+      - name: network.acls
+      - name: network.ospf
+    bug_specific_repos:
+      - name: network.backup
+
 triager:
-  email: testtriager@triage.example
-  password: superSecretP@ssWorD
-timedelta: 7
+  address: ${EMAIL_SENDER}
+  password: ${EMAIL_PASSWORD}
+
+timedelta: 14
+
+#github_token: ${GITHUB_TOKEN}  #Use GITHUB_TOKEN if you have a GitHub Actions workflow configured, otherwise comment this line out for unauthenticated requests."
+
+maintainers: ${MAINTAINERS}
+
+# github_token: 'Manually enter your github token here to make authenticated requests or comment this line to make unauthenticated API requests'
+
+
+
+
+##Generic structure 
```

Comment:
Strip this data out make example-config and add comments as explanation, 
Make this file as config.yaml should be in form of a template.

Reply:
Sure will do it thanks

Reply:
updated in new commit